### ISSUE TITLE
Variants fail to show

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/clin.iobio#384]
I have an analysis in https://mosaic.chpc.utah.edu/#/projects/521/analyses. If I launch the analysis, the variants don't show:

![Screen Shot 2021-03-18 at 12 11 37 PM](https://user-images.githubusercontent.com/523111/111659036-4206cd00-87e3-11eb-9f1e-0e63e7f0c654.png)

If I choose a different variant from the left menu, it explicitly tells me there are no variants in that gene:

![Screen Shot 2021-03-18 at 12 11 51 PM](https://user-images.githubusercontent.com/523111/111659085-4df28f00-87e3-11eb-9a76-4371b3d50101.png)

**Comment by tonydisera** [comment]
Yes, I'm seeing this problem as well.  The first variant that is clicked on (from the Variants tab in the side panel) doesn't load in the Variants card.  If I select another variant and then come back to the original variant, the variants load for the gene.
